pkgs/applications/virtualization/xen/generic/default.nix

```diff
@@ -323,6 +323,7 @@ stdenv.mkDerivation (finalAttrs: {
     "doc" # The full Xen documentation in HTML format.
     "dev" # Development headers.
     "boot" # xen.gz kernel, policy file if Flask is enabled, xen.efi if EFI is enabled.
+    # TODO: Python package to be in separate output/package.
```

Comment:
What exactly should the python output have? Since we already have a custom `installPhase`, just copying the python stuff to `$python` shouldn't be difficult. I'm not exactly sure what `python3Packages.xen` produces that `xen` doesn't.

Reply:
```shell
mkdir -p $python/lib
mv $out/lib/python* $python/lib/
```

Might be enough for python output, but I haven't tested that yet.

Reply:
I would prefer to resolve all of those TODOs after more progress on Qubes has made, and leave it TODO for now.

Reply:
Fine by me.